packages/components/src/Button/button.spec.tsx

```diff
@@ -1,31 +1,30 @@
-import { generateSnapshots, prepareStory } from "@chanzuckerberg/story-utils";
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { composeStories } from "@storybook/testing-react";
 import { render, screen } from "@testing-library/react";
 import React from "react";
 import Button from "./button";
 import * as ButtonStoryFile from "./button.stories";
 
+const { Default } = composeStories(ButtonStoryFile);
+
 describe("<Button />", () => {
   generateSnapshots(ButtonStoryFile);
 
   it("passes test ids down properly", () => {
-    render(
-      <Button data-testid="example-test-id">Button with data-testid</Button>,
-    );
+    render(<Default data-testid="example-test-id" />);
     expect(screen.getByTestId("example-test-id")).toMatchSnapshot();
   });
 
   it("passes class names down properly", () => {
     render(
-      <Button className="exampleClassName" data-testid="example-class-name">
-        Button with example className
-      </Button>,
+      <Default className="exampleClassName" data-testid="example-class-name" />,
     );
     expect(screen.getByTestId("example-class-name")).toMatchSnapshot();
   });
 
   it("forwards refs", () => {
     const ref = React.createRef<HTMLButtonElement>();
-    render(prepareStory(ButtonStoryFile.Default, { ref }));
+    render(<Button ref={ref}>Hi</Button>);
```

Comment:
Unfortunately, @storybook/testing-react doesn't let us pass refs to `React.forwardRef` components.

This kind of makes sense, because we're rendering the **_story_** (which isn't a forwardRef), not the **_component_** (which is).

But still kind of annoying. Working around for now by rendering Button instead of a story.